app/views/admin/projects.html.erb

```diff
@@ -31,12 +31,14 @@ See doc/COPYRIGHT.rdoc for more details.
   <%= render :partial => 'no_data' if @no_configuration_data %>
 </div>
 
-<% content_for :action_menu_specific do %>
-  <%= link_to l(:label_project_new), {:controller => '/projects', :action => 'new'}, :class => 'icon icon-add' %>
-<% end %>
 <% html_title(l(:label_administration), l(:label_project_plural)) -%>
-<h2><%=l(:label_project_plural)%></h2>
-<%= render :partial => 'layouts/action_menu_specific' %>
+<%= toolbar title: l(:label_project_plural) do %>
+  <li class="toolbar-item">
+    <%= link_to new_project_path, class: 'button -alt-highlight' do %>
+      <i class="button--icon icon-add"></i> <%= l(:label_project_new) %>
```

Comment:
I will probably refactor this in as i will have to write some sort of additional `toolbar_item` helper to scope under the `toolbar` method in `4.1.1`